scripts/release/homebrew/docker/formula_generate.py

```diff
@@ -126,11 +129,6 @@ def update_formula() -> str:
     upstream_sha = compute_sha256(HOMEBREW_UPSTREAM_URL)
     text = re.sub('sha256 ".*"', 'sha256 "{}"'.format(upstream_sha), text, 1)
     text = re.sub('.*revision.*\n', '', text, 1)  # remove revision for previous version if exists
-
-    # include pip when creating venv, see https://github.com/Homebrew/brew/pull/15792
-    # this can be removed after Homebrew merges our PR
-    text = re.sub('system_site_packages: false', 'system_site_packages: false, without_pip: false', text, 1)
-
```

Comment:
This is introduced in https://github.com/Azure/azure-cli/pull/27186, remove it as latest formula contains this change.